meilisearch-http/src/main.rs

```diff
@@ -27,6 +36,12 @@ async fn main() -> Result<(), MainError> {
                         .into(),
                 );
             }
+
+            #[cfg(feature = "sentry")]
+            if !opt.no_analytics {
+                sentry::integrations::panic::register_panic_handler();
+                sentry::integrations::env_logger::init(None, Default::default());
+            }
```

Comment:
`cfgs` works on if statements? Is it real?

Reply:
I ran it with the `--no-default-features` and without, and in both cases, it works as expected

Reply:
Hum, waht version of rust are you using?

Reply:
1.43.0